CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 ## jsonmatch 0.3 / 2018-07-30
```

Comment:
Need to update that section ?

jsonmatch 0.4 / today's date

Reply:
🤦‍♂good catch.